### DISCUSSION TITLE
Whether a Occlum instance directory can hold two executable binarys to run?

### DISCUSSION THREAD

**Discussion by lwxiaowuge** [occlum/occlum#847] [Q&A]
1，Does a Occlum instance directory correspond to only one binary, or does a Occlum instance directory contain multiple binarys? If yes, do they all belong to the security zone?
2，If a Occlum instance directory exist two binarys,Can the host OS program call occlum_pal_exec() to execute the two binary files outside?
3，What other ways do the outer enclave programs communicate with the Occlum program besides sharing memory?

**Comment by guzongmin** [comment]
I have added https://github.com/occlum/occlum/wiki/Q&A page to answer those questions.
